<ui version="4.0" >
 <class>InstallWindowFrm</class>
 <widget class="QDialog" name="InstallWindowFrm" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Install Rockbox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item rowspan="9" row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="rbutilqt.qrc" >:/icons/wizard.xpm</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Please select the Rockbox version you want to install on your player:</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Version</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="radioStable" >
        <property name="text" >
         <string>Rockbox &amp;stable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioArchived" >
        <property name="text" >
         <string>&amp;Archived Build</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCurrent" >
        <property name="text" >
         <string>&amp;Current Build</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QLabel" name="labelDetails" >
        <property name="text" >
         <string>Details about the selected version</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelNote" >
        <property name="text" >
         <string>Note</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="2" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;Install</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >
         <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >
         <normaloff>:/icons/process-stop.png</normaloff>:/icons/process-stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="QGroupBox" name="Backupgroup" >
     <property name="title" >
      <string>Backup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="backup" >
        <property name="text" >
         <string>Backup before installing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="backupLocation" >
        <property name="text" >
         <string>Backup location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="changeBackup" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Change</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QCheckBox" name="checkBoxCache" >
     <property name="toolTip" >
      <string>Rockbox Utility stores copies of Rockbox it has downloaded on the local hard disk to save network traffic. If your local copy is no longer working, tick this box to download a fresh copy.</string>
     </property>
     <property name="text" >
      <string>&amp;Don't use locally cached copy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rbutilqt.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>InstallWindowFrm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>562</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel" >
     <x>500</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>InstallWindowFrm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>658</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel" >
     <x>611</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
